drake/core/Path.h.in

```diff
@@ -2,7 +2,7 @@
 
 #include <string>
 
-namespace Drake {
+namespace {
```

Comment:
I think the motivation was the opposite.. perhaps it felt redundant to reference Drake::getDrakePath().  Thank you for the style policing.  It's going to take me time to learn the new style guide.